packages/xrpl/tools/generateModels.js

```diff
@@ -0,0 +1,265 @@
+const fs = require('fs')
+const path = require('path')
+const createValidate = require('./createValidate')
+const createValidateTests = require('./createValidateTests')
+
+function readFile(filename) {
+  return fs.readFileSync(filename, 'utf-8')
+}
+
+let jsTransactionFile
+
+function processRippledSource(folder) {
+  const sfieldCpp = readFile(
+    path.join(folder, 'src/ripple/protocol/impl/SField.cpp'),
```

Comment:
Can you add a comment explaining the setup you need to do before running this tool? 
(Ex. Do you need to build rippled, or just clone it?)

Reply:
Just clone it - it just needs access to the source code.